src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -1343,7 +1343,7 @@ export RADICAL_PILOT_VERBOSE=DEBUG
 $PREBOOTSTRAP2_EXPANDED
 
 # start agent, forward arguments
-$AGENT_CMD "\$1" 1>"\$1.out" 2>"\$1.err"
+exec $AGENT_CMD "\$1" 1>"\$1.out" 2>"\$1.err"
```

Comment:
How does it relate to the proctitle? And does it only relate to the proctitle or does it have other side-effects too?